### DISCUSSION TITLE
How work with events to normal livewire component

### DISCUSSION THREAD

**Discussion by YnievesDotNet** [Power-Components/livewire-powergrid#1381] [General]
Hi everyone, I like send an event dispatch to one livewire component, but don't work, or something I are passing and fail the process.

In powerGrid i make a button for create a PDF with a list of checked row, but don't download the pdf or stream the file using dompdf.

I try to build the pdf on a normal livewire component but is not launched the event 

I'm are using Laravel 10, livewire 3 and power grid 5

Thanks

**Comment by luanfreitasdev** [comment]
Hello. Can you give more details, example code ?

**Reply by YnievesDotNet** [reply]
https://github.com/Power-Components/livewire-powergrid/discussions/1224#discussioncomment-8361519

resolve whit that.

Thanks Luan

**Reply by YnievesDotNet** [reply]
I make a new component to make a downloadable pdf file.
